astrix-versioning/src/main/java/com/avanza/astrix/versioning/jackson2/VersionedJsonObjectMapper.java

```diff
@@ -253,7 +254,7 @@ private ObjectMapper buildMigratingMapper(ObjectMapper rawMapper, ThreadLocal<In
 		}
 
 		private ObjectMapper buildRaw() {
-			SimpleModule rawModule = new SimpleModule("Astrix-rawModule", new Version(1,0,0, ""));
+			SimpleModule rawModule = new SimpleModule("Astrix-rawModule", new Version(1,0,0, "", "", ""));
```

Comment:
The deprecated `Version` ctor uses `null` for `groupId` and `artifactId` instead of `""`.

Reply:
.. but internally , `null` and `""` are handled identically, so I guess it does not matter anyway.

Reply:
Changed it to null just to be safe